fedimint-client/src/backup.rs

```diff
@@ -135,7 +135,7 @@ impl EncryptedClientBackup {
     }
 }
 
-impl Client {
+impl ClientArc {
```

Comment:
In the PR description, it says

> For this to not require duplicating forwarding methods manually,
the inner ClientInner should just do as much as possible.
So in thie change I'm moving all methods to ClientInner,
making it more usable.

Shouldn't we also move all these backup-related methods on to `ClientInner`?

Reply:
Which methods exactly? Did I overlooked anything? Generally yes, everything should be on a new `Client`.